rotkehlchen/api/websockets/typedefs.py

```diff
@@ -15,7 +15,7 @@ class WSMessageType(Enum):
     PREMIUM_STATUS_UPDATE = auto()
     DB_UPGRADE_STATUS = auto()
     # Used for evm address migration after new chain integration
-    EVM_ADDRESS_MIGRATION = auto()
+    NEW_EVM_ACCOUNTS_DETECTED = auto()
```

Comment:
re consistency as per the previous comment

```suggestion
    EVM_ACCOUNTS_DETECTION = auto()
```